### PR TITLE
Fix sales admin member check - part 2

### PR DESCRIPTION
**Pull request by se-bastiaan**
Closes no issue

### Summary
Fix sales admin member check - part 2

### How to test
Steps to test the changes you made:
1. Go to the admin

### CODE REVIEW

**Comment by DeD1rk** [comment]
deploy review

File: website/sales/admin/order_admin.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def formfield_for_dbfield(self, db_field, request, obj=None, **kwargs):
             )
         if db_field.name == "shift":
             field.queryset = Shift.objects.filter(locked=False)
-            if request.member and not (
+            if not request.member or not (
                 (
```
Comment (DeD1rk) [inline website/sales/admin/order_admin.py:311]: These parentheses are not needed, same in line 323.

File: website/sales/admin/shift_admin.py [hunk 2]
```diff
@@ -132,7 +132,8 @@ def get_queryset(self, request):
         queryset = super().get_queryset(request)
 
         if not (
-            request.member.is_superuser
+            request.member
+            or request.member.is_superuser
```
Comment (DeD1rk) [inline website/sales/admin/shift_admin.py:136]: I think this is wrong: if there is a member, then the entire disjunction holds, so the `request.member` should be moved outside of the parentheses: `not request.member or not (request.member.is_superuser or request.member.has_perm(...))` like you used in `order_admin.py`. Same in line 174.

File: website/sales/admin/shift_admin.py [hunk 3]
```diff
@@ -170,7 +170,8 @@ def has_delete_permission(self, request, obj=None):
 
     def changelist_view(self, request, extra_context=None):
         if not (
```
Comment (DeD1rk) [inline website/sales/admin/shift_admin.py:172]: This one is still there

**Comment by DeD1rk** [comment]
Noo 😭 it needs or, not and. This way it still checks the superuser on no member

File: website/sales/admin/shift_admin.py [hunk 4]
```diff
@@ -169,7 +169,7 @@ def has_delete_permission(self, request, obj=None):
         return super().has_delete_permission(request, obj)
 
     def changelist_view(self, request, extra_context=None):
-        if not (
+        if not request.member and not (
```
Comment (DeD1rk) [inline website/sales/admin/shift_admin.py:172]:
```suggestion
        if not request.member or not (
```

**Comment by se-bastiaan** [comment]
> Noo 😭 it needs or, not and. This way it still checks the superuser on no member

😬

**Review by DeD1rk — APPROVED** [review]

**Pull request merged**